telemetry/specs/INT.mdk

```diff
@@ -306,25 +309,27 @@ at the monitoring system to collate reports from multiple devices.
 
 ## INT Applied to Synthetic Traffic
 
-In this category, INT Source devices generate synthetic traffic either by
-clonning the original data packets or by generating special probe packets.
-INT is applied to the synthetic traffic.
+INT Source devices may generate INT-marked synthetic traffic either by
+cloning  original data packets or by generating special probe packets.
+INT is applied to this traffic by transit nodes in exactly the same way 
+as all traffic. 
+
 
-The XD mode doesn't make sense for synthetic traffic. The MX mode is
-technically possible but doesn't add much value. Hence we define two MD modes
-as follows.
+The only difference between live traffic and Synthetic traffic is that 
+INT-Sink nodes may need to discard synthetic traffic after extracting 
+the collected INT data as opposed to forwading the traffic. This is 
+indicated by using the 'C' bit of the INT-Header to mark relavant packets
+as being Copies/Clones, to be discarded at the INT-Sink.  
 
-* INT-CLONE-MD (eMbed Data): INT Source devices create extra traffic through
-cloning the original packets and then embedding the INT instructions and metadata into
-the clones only. INT Transit devices accumulate metadata only on the clones,
-INT Sink devices may send telemetry reports before discarding the clones.
-Compared to INT-MD, INT-CLONE-MD produces extra traffic
-and may not capture actual network state that the orignal packets experience, while
-this mode avoids direct modification on the original data packets.
-This mode is similar to IFA [^IFA].
+All INT modes may be used on these Syntehtic/probe packets, as decided by the INT-Source node. 
+Specifically the **INT-MD** (eMbed Data) mode applied to Synthetic or probe packets allows 
+functionality similar to **IFA**[^IFA] 
 
-* INT-PROBE-MD (eMbed Data): similar to INT-CLONE-MD, but INT is applied to
-special probe packets instead of clone packets.
+It is likely that sytnetic traffic created by cloning would be discraded at the Sink,
```

Comment:
typos:
s/sytnetic/synthetic
s/discraded/discarded

Reply:
Corrected the typos.